src/Confluent.Kafka/Config.cs

```diff
@@ -0,0 +1,903 @@
+// *** Auto-generated *** - do not modify manually.
+//
+// Copyright 2018 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///     BrokerAddressFamily enum values
+    /// </summary>
+    public enum BrokerAddressFamilyType
+    {
+        /// <summary>
+        ///     Any
+        /// </summary>
+        Any,
+
+        /// <summary>
+        ///     V4
+        /// </summary>
+        V4,
+
+        /// <summary>
+        ///     V6
+        /// </summary>
+        V6
+    }
+
+    /// <summary>
+    ///     SecurityProtocol enum values
+    /// </summary>
+    public enum SecurityProtocolType
+    {
+        /// <summary>
+        ///     Plaintext
+        /// </summary>
+        Plaintext,
+
+        /// <summary>
+        ///     Ssl
```

Comment:
Is this decapalization standard in .NET?

Reply:
As part of a longer name for sure. Maybe not for an acronym in isolation. Added to #614 as a todo to check